leapp/libraries/stdlib/__init__.py

```diff
@@ -30,3 +40,31 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd, buffer):
+    if os.getenv('LEAPP_DEBUG', '0') == '1':
+        if fd == call.STDOUT:
+            sys.stdout.write(buffer)
+        else:
+            sys.stderr.write(buffer)
+
+
+def new_call(args, split=False):
+    _id = str(uuid.uuid4())
+    result_data = None
+    try:
+        create_audit_entry('process-start', {'id': _id, 'parameters': args})
+        result_data = call._call(args, callback_raw=_logging_handler)
+    finally:
+        create_audit_entry('process-end', _id)
+        create_audit_entry('process-result', {'id': _id, 'parameters': args, 'result': result_data})
```

Comment:
@vinzenz If `result_data` is buffer/bytes via: https://github.com/oamg/leapp/blob/5d1ee15732f5be216eab60560f9682a5719ec4ca/leapp/libraries/stdlib/call.py#L164 can the Audit serializer deal with that?

Reply:
AFAIK yes because it ahould be JSON serializable and that’s all that is required